src/lib.rs

```diff
@@ -266,3 +266,71 @@ macro_rules! declare_types {
 
     { } => { };
 }
+
+#[test]
+fn cli_test() {
+    use std::process::Command;
+    let cli_output = if cfg!(target_os = "windows") {
+        Command::new("cmd")
+                .args(&["/C", "Implement this later"])
+                .output()
+                .expect("failed to execute process")
+    } else {
+        Command::new("sh")
+                .arg("-c")
+                .arg("cd cli; npm install; npm run transpile")
```

Comment:
I think it'd be better to chain these with `&&` instead of `;` so that the whole command fails if any of the individual steps fails.